bank2ynab/config_handler.py

```diff
@@ -101,7 +101,7 @@ def fix_conf_params(self, section: str) -> dict[str, Any]:
             ),
             "plugin": self.get_config_line_str(section, "Plugin"),
             "plugin_args": self.get_config_line_lst(
-                section, "Plugin Arguments", ","
+                section, "Plugin Arguments", "\n"
```

Comment:
No plugin uses the `Plugin Arguments` option yet, so this seemed safe to update.